internal/pkg/scalable/cronjobs.go

```diff
@@ -29,6 +28,18 @@ type cronJob struct {
 	*batch.CronJob
 }
 
+// Reget regets the resource from the Kubernetes API.
+func (c *cronJob) Reget(clientsets *Clientsets, ctx context.Context) error {
+	singleCronJob, err := clientsets.Kubernetes.BatchV1().CronJobs(c.Namespace).Get(ctx, c.Name, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Errorf("failed to get cronjob: %w", err)
+	}
+
+	c.CronJob = singleCronJob
+
+	return nil
```

Comment:
```suggestion
    var err error
	c.CronJob, err = clientsets.Kubernetes.BatchV1().CronJobs(c.Namespace).Get(ctx, c.Name, metav1.GetOptions{})
	if err != nil {
		return fmt.Errorf("failed to get cronjob: %w", err)
	}

	return nil
```

this should be done with every resource